leesy/index/aboutme.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <meta charset="UTF-8">
+    <title> leesy ruiz </title>
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/aboutme.css" rel="stylesheet" type="text/css">
+    <link href="/Users/lisyruiz/Documents/Ada/6_week/Static-Site/leesy/style/normalize.css" rel="stylesheet" type="text/css">
```

Comment:
You should be using relative paths here rather than absolute ones. An absolute path only applies on your computer, whereas a relative one will work for anyone who has checked out your project.